WalletWasabi/Blockchain/TransactionBuilding/BnB/CoinSelection.cs

```diff
@@ -8,6 +8,8 @@ namespace WalletWasabi.Blockchain.TransactionBuilding.BnB;
 /// </summary>
 public class CoinSelection
 {
+	internal static readonly int MaxCoinCount = 5;
```

Comment:
Please adjust this to the originally built transaction. For example if the originally build tx had 5 coins, then this could be 1.5x5 coins = 7 or 8.